src/crates_cache.rs

```diff
@@ -147,17 +155,34 @@ impl CratesCache {
 
         // Not modified.
         if response.status() == 304 {
+            bar.finish_and_clear();
             return Ok(DownloadState::Fresh);
         }
 
+        if let Some(length) = response
+            .header("content-length")
+            .and_then(|l| l.parse().ok())
+        {
+            bar.set_style(
+                indicatif::ProgressStyle::default_bar()
+                    .template("{prefix:>12.bright.cyan} [{bar:27}] {bytes:>9}/{total_bytes:9}  {bytes_per_sec} {elapsed:>4}/{eta:4} - {msg:.cyan}")
```

Comment:
`{eta:4}` seems to print the remaining time, in the `elapsed/eta` format I find it a bit confusing. Maybe print `elapsed/total` or just the ETA?

Reply:
For reference, `wget` prints just the ETA